handlers.py

```diff
@@ -30,7 +30,22 @@ class Status:
 def do(message):
     # mark task state as DO (in progress)
     # assign from_user id as assignee of the task
-    pass
+    _, index = message.text.strip().split()
+    index = int(index) - 1
+    task = r.lindex(f'/tasks/chat_id/{message.chat.id}', index)
+    if task is None:
+        return bot.reply_to(message, 'No task with such id.')
+    task = json.loads(task.decode())
+    task['status'] = Status.DO
```

Comment:
Also, change assignee field to user.id
Probably we should add assignee Name as well, what do you think?